board/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+
+from user.models    import User
+from product.models import Product
+
+
+class Review(models.Model):
+    author           = models.ForeignKey(User, on_delete=models.SET_DEFAULT, default=1)
+    product          = models.ForeignKey(Product, on_delete=models.SET_DEFAULT, default=1)
+    title            = models.CharField(max_length=200)
+    contents         = models.TextField()
+    help_count       = models.IntegerField()
+    hit_count        = models.IntegerField()
+    created_at       = models.DateTimeField(auto_now_add=True)
+    updated_at       = models.DateField(auto_now=True)
+    review_image_url = models.URLField(max_length=2000)
```

Comment:
Review instance의 image_url 속성에 접근할 경우, 아래와 같은 이유로 가독성을 해칠 수 있습니다.
```python
review = Review.objects.get()
(기존) review.review_image_url
(수정) review.image_url
```
```suggestion
    image_url = models.URLField(max_length=2000)
```

Reply:
이름 수정하도록 하겠습니다.